### DISCUSSION TITLE
New receiver does not add new entry on feet or logstash | posible bug on 2.4.80

### DISCUSSION THREAD

**Discussion by Wilkins00** [Security-Onion-Solutions/securityonion#13582] [2.4]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

32

### RAM

64

### Storage for /

200

### Storage for /nsm

500

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi Security onion team, 

Recently I am having a probable bug , I added a new receiver and the entry on setting to start load balancing **the manager log from the agent** is not added it should add the entry to receive the log in case the manager go down.
![image](https://github.com/user-attachments/assets/2ebff793-d0af-410c-aa69-73cd6e13d041)
**the section output**  also is server is not added which might be a bug , I need to know how to add it manually please.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
Does this new receiver show up under 'Grid' in SOC and have all services reporting their status as 'OK'?

You can try running ```sudo so-checkin``` on your new receiver and see if it reports any errors that you can share here

**Comment by Wilkins00** [comment]
Hello,
I rebuild everything and the any was created, you can proceed and close this case.
